PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Holder/MegFileDataEntry.cs

```diff
@@ -1,51 +1,73 @@
+// Copyright (c) 2021 Alamo Engine Tools and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
 using System;
 
 namespace PG.StarWarsGame.Files.MEG.Holder
 {
     /// <summary>
-    /// An object representing a file packaged in a *.MEG file.
+    ///     An object representing a file packaged in a *.MEG file.
     /// </summary>
     [Serializable]
```

Comment:
why?

Reply:
Because all holders are supposed to be `[Serializable]` but for some reason that attribute is not inherited (see: https://stackoverflow.com/a/183149). Same goes for exceptions. Technically the base class has the annotation, but you have to have it on each inheriting class anyways.

Reply:
i guess for some very good reason the attribute is not inherrited. Some say Serialization is evil :) 
The question was more like: Why do want to serialize holders? Am I missing something here? Because if there is no demanding feature request to it, i'd rather not do that.